2.3/Dockerfile.rhel7

```diff
@@ -0,0 +1,41 @@
+FROM rhscl/s2i-base-rhel7
+
+# This image provides a Ruby 2.3 environment you can use to run your Ruby
+# applications.
+
+EXPOSE 8080
+
+ENV RUBY_VERSION 2.3
+
+LABEL summary="Platform for building and running Ruby 2.3 applications" \
+      io.k8s.description="Platform for building and running Ruby 2.3 applications" \
+      io.k8s.display-name="Ruby 2.3" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,ruby,ruby23,rh-ruby23"
+
+# Labels consumed by Red Hat build service
+LABEL BZComponent="rh-ruby23-docker" \
+      Name="rhscl_beta/ruby-23-rhel7" \
+      Version="2.3" \
+      Release="3" \
+      Architecture="x86_64"
+
+RUN yum-config-manager --enable rhel-server-rhscl-7-rpms && \
```

Comment:
At this point there is only Beta available so we need to have `rhel-server-rhscl-7-beta-rpms` enabled instead.